server/conf/docker/docker-compose-dev.all.debug.yml

```diff
@@ -125,7 +125,7 @@ services:
     container_name: core_portal_workers
 
   docs:
-    image: taccwma/frontera-docs:7af12ae
+    image: taccwma/frontera-docs:aea2e4f
```

Comment:
If the CMS baked in the theme variable values, then this is **not** necessary, because User Guides would have _zero_ changes to make.

Reply:
That's the plan with https://github.com/TACC/Core-CMS/pull/202.